### DISCUSSION TITLE
commit / merge text

### DISCUSSION THREAD

**Discussion by lscheffler** [VFPX/Thor#139] [General]
Could the text of merges / commits be more descriptive
- what is fixed or changed?

pls

**Comment by Jimrnelson** [comment]
I am not going to describe a one or two-line bug fix.  Description of what is actually fixed is best understood by looking at the code.

**Comment by lscheffler** [comment]
This is called _good practice_ in the open source world. I know it's a pain (I'm from the _comments? read the code, it's there_ tribe as well), but it has value. In special if one works around the world and over language barriers.   
In case as simple as _Fixing #137_, one can follow the link, and read there. (btw, this should close the issue)
